protocol/simplified-guardian.md

```diff
@@ -0,0 +1,121 @@
+clusters, it may be valuable to implement a Cluster-wide Pause on top of the Superchain-wide Pause.
+
+A chain would be paused if *either* the Superchain-wide Pause is active or its Cluster-wide Pause
+is active. We would implement the Cluster-wide Pause by allowing the Guardian to pause a specific
+chain/cluster as identified by a `ETHLockbox` contract address. That is, the Guardian would call
+`SuperchainConfig.pause(ethLockboxAddress)`. Other contracts would then check the status of the
+pause by calling `SuperchainConfig.paused(myEthLockboxAddress)` which would return true if either
+that specific system is paused or the entire Superchain is paused.
+
+This has the advantage of guaranteeing that a cluster of chains that share a bridge and proof
+system can easily be paused together as a result of their shared `ETHLockbox` address. Exact
+details of this mechanism will be specified later.
+
+## Considerations
+
+### Pause Expiry Security Implications
+
+Pause Expiry has non-trivial security implications in that a bug in the proof system alongside a 
+malicious blocking minority of the Security Council could be catastrophic. L2Beat explciitly argues
+that this security model is better than the model in which a malicious Foundation and a blocking
+minority of the Security Model is catastrophic. Since this is fundamental to the updated Stage 1
+definition, we are required to accept this conclusion. We may want to take this as motivation to
+implement some other proof system as a backup to the primary proof system.
+
+### Permissioned Proof System
+
+A chain will be considered Stage 0 if it falls back to the Permissioned Proof System. We should
+accept this potential for a downgrade and generally rely on the pause mechanism as the primary
+incident response tool.
+
+We propose the following incident response protocol:
+
+- Always attempt to resolve an issue offchain if possible, before a game resolves incorrectly. This
+  scenario would typically occur if the forecasted incorrect resolution is the result of an issue
+  with offchain infrastructure rather than a bug in smart contracts.
+- Determine if the issue is chain-specific (e.g., missing honest actor) or applies to all chains
+  (e.g., contract bug in dispute game).
+  - If chain-specific or cluster-specific, execute the chain-specific or cluster-specific pause.
+  - If applies to all chains, execute the Superchain-wide pause.
+- We immediately determine if we believe that the bug can be resolved within 2 weeks.
+- If the bug can be resolved within 2 weeks (via Security Council emergency upgrade) then we
+  keep the pause active until the bug is resolved, upgrade the game, or retire all existing games,
+  and unpause.
+- If the bug cannot be resolved within 2 weeks or if 2 weeks have actually passed since the pause
+  was executed then we switch to the Permissioned Proof System, or retire all existing games, and
+  unpause.
```

Comment:
Do we actually need to keep the permissioned game? Given that it's existence was the motivation for a lot of the design, can we simplify the design at all by removing permissioned games entirely? The result would just be the last bullet here is gone, and we maintain the pause for longer until the issue can be resolved

Reply:
Yes I would say we need to keep the permissioned game. If there were an issue found in the way the dispute game itself worked (vs just a bug in the implementation of it), it could take us a significant amount of time to design, implement and test a fix for it.  Having withdrawals completely disabled during that time would incur massive reputational damage and the time pressure to develop the fix would increase the risk of the fix being insufficient or introducing other problems.
I'd quite like to have time to simplify the permissioned game so it isn't a full dispute game implementation though.